inits.json

```diff
@@ -77,7 +77,10 @@
             "Star Metallicity (-) Uncertainty": -0.08,
             "Star Surface Gravity (log(g))": 4.22,
             "Star Surface Gravity (+) Uncertainty": 0.04,
-            "Star Surface Gravity (-) Uncertainty": -0.04
+            "Star Surface Gravity (-) Uncertainty": -0.04,
+            "Star Distance (pc)": 289.21,
+            "Star Proper Motion RA (mas/yr)": -9.82,
+            "Star Proper Motion DEC (mas/yr)": 3.48
     },
     "optional_info": {
```

Comment:
A comment here would not hurt to let readers know HAT-P-32 b is just a default reference. ...

Reply:
At the top of the file, it states that the initialization file is for HAT-P-32 b.

Reply:
Thanks, not visible in the diff snippet.